projects/chaos/test/test_power_path.c

```diff
@@ -4,18 +4,25 @@
 #include <stdint.h>
 
 #include "adc.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
 #include "chaos_events.h"
 #include "delay.h"
 #include "event_queue.h"
+#include "fsm.h"
 #include "gpio.h"
 #include "gpio_it.h"
 #include "interrupt.h"
 #include "log.h"
+#include "misc.h"
 #include "soft_timer.h"
 #include "status.h"
 #include "test_helpers.h"
 #include "unity.h"
 
+#define TEST_POWER_PATH_NUM_RX_HANDLERS 1
+
 #define TEST_POWER_PATH_ADC_PERIOD_US 1000
 
 #define TEST_POWER_PATH_AUX_CURRENT_VAL 1
```

Comment:
Could you maybe rename these `#define`s to make it a bit more clear what they represent (ie. whether it is an overvoltage or undervoltage value)? And maybe the same for your conversion functions that you're overriding.

Reply:
Done.